cql3/statements/alter_keyspace_statement.cc

```diff
@@ -84,11 +84,16 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
 future<std::tuple<::shared_ptr<cql_transport::event::schema_change>, std::vector<mutation>, cql3::cql_warnings_vec>>
 cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_processor& qp, const query_options&, api::timestamp_type ts) const {
     try {
-        auto old_ksm = qp.db().find_keyspace(_name).metadata();
+        auto ks = qp.db().find_keyspace(_name);
+        auto ks_md = ks.metadata();
```

Comment:
why change the variable name?
it creates extraneous churn...

Reply:
The reason was that I needed a separate variable for the ks object and a separate variable for keyspace_metadata object. Secondly, the orig var was named incorrectly, because it was referring to ks metadata, and not the ks itself.
But I reverted this change already, so the conversation can be resolved.